src/app/components/register/register.component.spec.ts

```diff
@@ -19,6 +19,8 @@ import { NavBarComponent } from '../nav-bar/nav-bar.component';
 import { SingleHeroPageComponent } from './../single-hero-page/single-hero-page.component';
```

Comment:
you could just neuter the .spec file as in other places. Pls follow the company policy.

Reply:
solved